.gitignore

```diff
@@ -1,6 +1,4 @@
 node_modules
 coverage
-todo.txt
 *.swp
 *.swo
-package-lock.json
```

Comment:
why was this one ignored in the first place?

Reply:
No idea why. It was introduced [here](https://github.com/publiclab/Leaflet.DistortableImage/commit/e24f13647ea615300c5ca8dd8b5d99f1b237b2e9) if you wish to see the blame.